node-definitions/arista/veos/vEOS.yaml

```diff
@@ -0,0 +1,64 @@
+id: vEOS
+general:
+  description: Arista vEOS
+  nature: switch
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - eth0
+      - eth2
+      - eth3
+      - eth4
+      - eth5
+      - eth6
+      - eth7
+      - eth9
+      - eth10
+      - eth11
+      - eth12
+      - eth13
+    serial_ports: 1
+    default_count: 13
```

Comment:
This allocates all 13 interfaces by default (which may be fine and required).  But generally, we recommend providing a smaller set of default interfaces if you can.

Reply:
I can adjust to 5(incl. mgmt), I generally like lots of default interfaces for switches.

Reply:
I can adjust to 5(incl. mgmt). I generally like a lot of default interfaces for switches

Reply:
I was just asking.  It could be said that the full up-front interface allocation is preferred if it doesn't otherwise cause a memory or CPU issue with the device.